ocaml/xapi/storage_migrate.ml

```diff
@@ -670,7 +670,12 @@ let receive_start ~dbg ~sr ~vdi_info ~id ~similar =
 			| Some vdi ->
 				debug "Cloning VDI %s" vdi.vdi;
 				let vdi = add_to_sm_config vdi "base_mirror" id in
-				Local.VDI.clone ~dbg ~sr ~vdi_info:vdi
+				let c_vdi = Local.VDI.clone ~dbg ~sr ~vdi_info:vdi in
```

Comment:
Hi @lindig, naming updated. Though I used `vdi_clone`  to replace `c_vdi` as `clone` is used as function/verb in many places, also used `new_size` to replace `c_size`.